tests/unit/test_tb_x86.c

```diff
@@ -245,8 +245,9 @@ static void test_tb_x86_64_32_imul_Gv_Ev_Ib(void **state)
     int64_t edi = 0x488ac239;
 #endif
 
-    mem = calloc(1, CODE_SPACE);
-    assert_int_not_equal(0, mem);
+    //mem = calloc(1, CODE_SPACE);
+    // TODO examine
+    //assert_int_not_equal(0, mem);
 
```

Comment:
@danghvu referring to your comment on line 255, this assert is the issue. since `mem` is a `void*` and not an `int`, this doesn't compile. with `-Wall -Werror`, you then need to remove an unused variable